lib/datasets/dataset.rb

```diff
@@ -4,6 +4,7 @@
 require_relative "error"
 require_relative "metadata"
 require_relative "table"
+require_relative "cache-path"
```

Comment:
Could you keep this list in alphabetical order?

Reply:
OK